docs/operator-manual/argocd-cm.yaml

```diff
@@ -101,6 +102,11 @@ data:
     jsonPointers:
     - /spec/replicas
 
+  # Configuration to define customizations ignoring differences during watched resource updates to skip application reconciles.
+  resource.customizations.ignoreResourceUpdates.all: |
```

Comment:
Might also be good to have a resource-specific example, just so folks can be certain that it's an option.